modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view_{context}"]
+= Customizing a Developer catalog or its sub-catalogs using the form view
+
+You can customize a developer catalog by using the form view in the Web Console.
+
+.Prerequisites
+
+* An OpenShift web console session by a cluster administrator.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Administration* -> *Cluster Settings*.
+. Select the *Configuration* tab and click the *Console (operator.openshift.io)* resource.
+. Click *Actions* -> *Customize*.
+. In the respective section, enable or disable the items in the list.
```

Comment:
Suggestion: "In the respective sections" or "In the appropriate section" - respective sounds off in the singular

Reply:
Nice catch, it does sound singular. Fixed.